federation/registry/cluster/strategy_test.go

```diff
@@ -27,6 +27,7 @@ import (
 	genericapirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/kubernetes/federation/apis/federation"
 	"k8s.io/kubernetes/pkg/api"
+	_ "k8s.io/kubernetes/pkg/api/testapi"
```

Comment:
Oh, `testing` and `testapi` read nearly the same. Makes sense then.

Can you separate that import line somehow like this:
```
...
apitesting "k8s.io/kubernetes/pkg/api/testing"

// install all api groups for testing
_ "k8s.io/kubernetes/pkg/api/testapi"
```